source/funkin/backend/scripting/Script.hx

```diff
@@ -194,6 +205,8 @@ class Script extends FlxBasic implements IFlxDestroyable {
 		this.path = path;
 		onCreate(path);
 		for(k=>e in getDefaultVariables(this)) {
+			if(getBlacklistedImports().contains(k))
```

Comment:
Id recommend calling this once just outside of the loop

Reply:
Do you mean creating a separate array to use it or create a map from getDefaultVariables and modify it?

Reply:
Nah just like

var blockedImports = Script.getDefaultImports()
for k >= in aaaaaaa:
Im tooo tired rn but you get me